lib/manager/components/ProjectFeedListToolbar.js

```diff
@@ -288,6 +307,46 @@ export default class ProjectFeedListToolbar extends PureComponent<Props> {
     return options
   }
 
+  _renderLabelFilter = () => {
+    const {
+      project,
+      filter
+    } = this.props
+    // This variable must be set directly from this.props both
+    // to allow the override and for flow
+    const labels = this.props.filter.labels || []
+    const { labelsFilterMode } = filter
+
+    return (
+      <MenuItem header style={{ minWidth: 275 }}>
+        <h5>
+          Show{' '}
+          <select
+            name='anyonly'
+            id='anyonly'
+            value={labelsFilterMode}
+            onChange={this._onLabelModeSelectorClick}
+          >
+            <option value='any'>any</option>
+            <option value='only'>only</option>
+          </select>{' '}
+          feeds with the labels
```

Comment:
As a reader, this message doesn't make a ton of sense to me. I would change to something like `Show feeds containing [any|all] of the selected labels:`. 

Thoughts?

Reply:
I personally prefer any/only, but I support the wisdom of the masses. Changed!